frontend/src/helpers/utils.js

```diff
@@ -105,34 +105,40 @@ const codesToDisplayText = {
         "team-member": "Team Members",
         "budget-line-items": "Budget Line Items"
     },
-    className: {
+    classNameLabels: {
```

Comment:
may consider moving all lookup helpers into its own module like `/helpers/lookup.helpers.js`

Reply:
hmmm, both API lookups and the convert display labels.  I might follow up on that.